flink-docs/src/test/java/org/apache/flink/docs/rest/RestAPIDocGeneratorTest.java

```diff
@@ -60,6 +62,22 @@ void testExcludeFromDocumentation() throws Exception {
                         "This REST API should also not appear in the generated documentation.");
     }
 
+    @Test
+    void testAdditionalFields() {
+        final String messageHtmlEntry =
+                RestAPIDocGenerator.createMessageHtmlEntry(
+                        TestAdditionalFields.class, null, EmptyRequestBody.class);
+        assertThat(messageHtmlEntry)
+                .isEqualTo(
+                        "{\n"
+                                + "  \"type\" : \"object\",\n"
+                                + "  \"id\" : \"urn:jsonschema:org:apache:flink:docs:rest:RestAPIDocGeneratorTest:TestAdditionalFields\",\n"
+                                + "  \"additionalProperties\" : {\n"
+                                + "    \"type\" : \"string\"\n"
+                                + "  }\n"
+                                + "}");
```

Comment:
nit: this assertion also seems a bit fragile because it includes formatting. Should we use something like `contains`?